alfresco-realm.json

```diff
@@ -1684,6 +1684,35 @@
       "account" : [ "manage-account", "view-profile" ]
     },
     "groups" : [ "/admin" ]
+  },{
+    "id" : "418b64ff-2ddd-46d5-820d-5979a37cc54b",
+    "createdTimestamp" : 1528478600816,
+    "username" : "admin",
+    "enabled" : true,
+    "totp" : false,
+    "emailVerified" : true,
+    "firstName" : "admin",
+    "lastName" : "admin",
+    "email" : "admin@app.activiti.com",
+    "credentials" : [ {
+      "type" : "password",
+      "hashedSaltedValue" : "+A2UrlK6T33IHVutjQj9k8S8kMco1IMnmCTngEg+PE+2vO4jJScux6wcltsRIYILv5ggcS3PI7tbsynq5u39sQ==",
```

Comment:
Looks like this was copied and pasted from testuser.  I think this needs to be a hashed password of `admin`.

Reply:
yes the hashed 'password' was copied :-), will update with new value